create/how_to/aws.md

```diff
@@ -53,7 +53,7 @@ Simply click on **Next: Add Storage** to keep going.
 
 Choose the storage **device** and **size** for your MeiliSearch instance.
 
-![Storage](/aws/05.storage.png)
+![Choosing the storage device and size](/aws/05.storage.png)
```

Comment:
After thinking about it more, even if it is default, it may still be helpful to note that in the image, General Purpose SSD (gp2) is selected as the Volume Type. It can still reassure a user that what they have selected is not wrong.